### PR TITLE
Fixes to make web 1.0.0 work

### PR DESCRIPTION
**Pull request by syleishere**
Some code fixes so web:^1.0.0 will work in pubspec.yaml for record_web

### CODE REVIEW

File: record_web/lib/js/js_import_library.dart [hunk 1]
```diff
@@ -23,7 +24,7 @@ class ImportJsLibrary {
       ..type = "text/javascript"
       ..charset = "utf-8"
       ..id = id
-      ..innerHTML = content;
+      ..innerHTML = js_util.jsify(content); // Use jsify to convert
```
Comment (llfbandit) [inline record_web/lib/js/js_import_library.dart:27]: Why do we need to jsify now? Is there an breaking change with 0.5.1?

File: record_web/pubspec.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
 
-  web: ^0.5.1
+  web: ^1.0.0
```
Comment (llfbandit) [inline record_web/pubspec.yaml:17]: I do prefer ">=0.5.1 <2.0.0" to not break lower bound. 0.5.1 should be compatible with 1.0.0 as far as I know.

**Review by llfbandit — CHANGES_REQUESTED** [review]

File: record_web/pubspec.yaml (hunk 2 above)
Comment (syleishere) [reply record_web/pubspec.yaml:17]: Soon as you enable web:^1.0.0, first error you will get is:

```
Target dart2js failed: ProcessException: Process exited abnormally with exit code 1:
../../../../AppData/Local/Pub/Cache/hosted/pub.dev/record_web-1.1.1/lib/js/js_import_library.dart:26:21:
Error: A value of type 'String' can't be assigned to a variable of type 'JSAny'.
      ..innerHTML = content;
                    ^
Error: Compilation failed.
```

This patches it for 1.0.0 stable on so package does not break.

Comment (llfbandit) [reply record_web/pubspec.yaml:17]: Hmm, OK

**Comment by llfbandit** [comment]
I made changes that should be compatible with web: 1.0.0.

Can you check it from this branch https://github.com/llfbandit/record/tree/web_10 ?
I can't upgrade Flutter for myself yet.

**Comment by syleishere** [comment]
Yeah sure I'll run a basic test on it tonight and get back to you.

**Comment by syleishere** [comment]
There were no compiler errors, looks good on that end.

**Comment by llfbandit** [comment]
web 1.1.2 has been released

**Pull request closed without merging**